omod/src/main/resources/apps/formentry_app.json

```diff
@@ -5,11 +5,13 @@
         "extensions": [
             {
                 "id": "formentryapp.forms.homepageLink",
-                "extensionPointId": "org.openmrs.referenceapplication.configuremetadataLink",
+ 				"extensionPointId": "org.openmrs.module.adminui.adminLinks",
                 "type": "link",
-                "label": "formentryapp.manageforms",
+                "label": "referenceapplication.formentryapp.manageform.label",
                 "url": "formentryapp/forms.page?app=formentryapp.forms",
-                "icon": "icon-paste",
+		        "extensionParams":{
+		            "group": "org.openmrs.module.adminui.manageforms.group"
```

Comment:
Does this group extension point exist? I think you need to add it? Did you test it in the UI to see if both the group and the link appear on the configure metadata page?